ingestion/src/metadata/utils/logger.py

```diff
@@ -128,6 +131,18 @@ def test_suite_logger():
     return logging.getLogger(Loggers.TEST_SUITE.value)
 
 
+def log_query(query) -> str:
```

Comment:
I wasn't able to find if this is used.

Reply:
Correct. i started with that and moved the listener. Let me remove it